pom.xml

```diff
@@ -608,7 +611,10 @@
             <!--Only @Plugin classes in the export packages will be included as plugin-->
             <_exportcontents>
               io.cdap.plugin.gcp.*;
+              com.google.auth.oauth2.*;
               com.google.cloud.hadoop.*;
+              com.google.pubsub.v1.*;
+              com.google.protobuf.*;
```

Comment:
Why do we need to export so many things?? Generally it is not good. If it is for spark streaming serialization, we should be able to hide most of them by being more careful about what goes into class fields and marking them transient when appropriate.

Reply:
@chtyim That's exactly what happened to me. Spark complains about missing classes if those 2 packages are not available.

Reply:
I'll try to clean this up next.